### PR TITLE
Update Prysm Helm Chart to 1.3.8 and Prater and update validator to validator client.

### PR DESCRIPTION
**Pull request by LaContra**
- Rename eth2prysm to helm.
- Update the repo name.
- Update the password file name in the yamls.
- Update Chart version.

### CODE REVIEW

File: prysm/nfs/validator-deployment.yaml [hunk 1]
```diff
@@ -58,7 +58,7 @@ spec:
           secretName: wallet-password-1
           items:
           - key: password
-            path: password.txt
+            path: wallet_pass.txt
```
Comment (chingyi-lin) [inline prysm/nfs/validator-deployment.yaml]: can we change it to `wallet_password.txt` or `wallet_pwd.txt`?

File: prysm/helm/templates/rbac.yaml [hunk 2]
```diff
@@ -21,7 +21,7 @@ rules:
   resources: ['podsecuritypolicies']
   verbs:     ['use']
 - apiGroups: [""]
-  resources: ["secrets", "configmaps"]
+  resources: ["secrets"]
```
Comment (chingyi-lin) [inline prysm/helm/templates/rbac.yaml:24]: thank you!

File: prysm/helm/values.yaml [hunk 3]
```diff
@@ -25,7 +25,7 @@ nfs:
   # All directories and files created with the processes are owned by this group.
   group: 2000
 
-# A required section that configures beacon chain and validator images registry and version.
+# A required section that configures beacon chain and validator client images registry and version.
```
Comment (chingyi-lin) [inline prysm/helm/values.yaml]:
```suggestion
# A required section that configures image registry, image name and version tag for beacon chain and validator client.
```

File: prysm/nfs/validator-deployment.yaml (hunk 1 above)
Comment (LaContra) [reply prysm/nfs/validator-deployment.yaml]: I changed both yamls and helm chart.

**Comment by LaContra** [comment]
@chingyi-lin I just updated the Chart version in this PR as well.

**Review by chingyi-lin — APPROVED** [review]

**Pull request merged**